renderer/nyaovim-app.ts

```diff
@@ -164,6 +164,117 @@ Polymer({
                     client.command(c);
                 }
             });
+
+            ipc.on('nyaovim:copy', (_: Electron.IpcRendererEvent) => {
+                // get current vim mode
+                var m = client.commandOutput('echo mode()');
+                m.then(value => {
+                    //  mode() returns a strange '\n' at the beginning, why?
+                    value = value.trim();
+                    if (value.length > 0) {
+                        const ch = value[0];
+                        if (ch == 'v'  // visual mode
+                            || ch == 'V' // visual line mode
+                            || (ch == '^' && value[1] == 'V') // visual block mode
+                           ) {
+                            const command = '"+y';
+                            client.input(command);
+                        } else {
+                            // execute the default copy command
+                            const webContents = ThisBrowserWindow.webContents;
+                            webContents.copy();
```

Comment:
I think default copy behavior (this else clause) is not needed because it's confusing.  Please simply ignore if Vim can't yank some text.